Assets/Scripts/Models/Character.cs

```diff
@@ -295,7 +290,10 @@ public void PrioritizeJob(Job job)
         DestTile = MyJob.tile;
 
         // If the dest tile does not have neighbours that are walkable it's very likable that they can't be walked to.
-        if (DestTile.GetNeighbours().Any((tile) => { return tile.MovementCost > 0; }) == false)
+        if (DestTile.GetNeighbours().Any((tile) =>
```

Comment:
Should we make a ```HasWalkableNeighbours()``` method in Tile and use it here, since this line is used twice in this file.